ydb/docs/en/core/dev/system-views.md

```diff
@@ -476,3 +476,87 @@ Table structure:
 |--------|-------------|
 | `Path` | Path to the access object.<br />Type: `Utf8`.<br />Key: `0`. |
 | `Sid` | SID of the access object owner.<br />Type: `Utf8`. |
+
+## Query sessions {#query-sessions}
+
+* `query_sessions`: Contains detailed information about currently active sessions.
+
+This view is useful for monitoring active database sessions, identifying long-running queries, tracking client connections, and analyzing session patterns.
+
+Table structure:
+
+| Column | Description | Data type |
+| --- | --- | --- |
+| `SessionId` | Unique session identifier.<br/>Key: `0`. | `Utf8` |
+| `NodeId` | Identifier of the node where the session is running. | `Uint32` |
+| `State` | Current state of the session (e.g., "IDLE", "EXECUTING"). | `Utf8` |
+| `Query` | Text of the currently executing query, if any. | `Utf8` |
+| `QueryCount` | Total number of queries executed in this session. | `Uint32` |
+| `ClientAddress` | IP address of the client connection. | `Utf8` |
+| `ClientPID` | Process ID information from the client. | `Utf8` |
+| `ClientUserAgent` | User agent string provided by the client. | `Utf8` |
+| `ClientSdkBuildInfo` | SDK build information from the client. | `Utf8` |
+| `ApplicationName` | Application name specified by the client. | `Utf8` |
+| `SessionStartAt` | Timestamp when the session was created. | `Timestamp` |
+| `QueryStartAt` | Timestamp when the current query started executing (if any). | `Timestamp` |
```

Comment:
This column is optional/nullable, then? What's the value when there's no currently executing query?